app/vep/model.py

```diff
@@ -0,0 +1,81 @@
+# generated by datamodel-codegen:
+#   filename:  APISpecification.yaml
+#   timestamp: 2024-06-10T15:52:34+00:00
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, List, Optional, Union
+
+from pydantic import BaseModel, Field
+
+class PaginationMetadata(BaseModel):
+    page: int
+    per_page: int
+    total: int
+
+class PredictedIntergenicConsequence(BaseModel):
+    feature_type: Any = Field(
+        ...,
+        description='The value of this field is always null. The presence of null in this field will serve as a marker that this is a consequence of an intergenic variant.',
+    )
+    consequences: List[str] = Field(
+        ...,
+        description="The only expected member of this array will be the string 'intergenic_variant'",
+    )
+
+
+class FeatureType(Enum):
+    transcript = 'transcript'
+
+
+class Strand(Enum):
+    forward = 'forward'
+    reverse = 'reverse'
+
+
+class PredictedTranscriptConsequence(BaseModel):
+    feature_type: FeatureType
+    stable_id: str = Field(..., description='transcript stable id, versioned')
+    gene_stable_id: str = Field(..., description='gene stable id, versioned')
+    gene_symbol: str
```

Comment:
How does the model make sure that a field is nullable (e.g. string or null) as opposed to non-nullable (e.g. only string)?

In the spec, we have a bunch of nullable fields, e.g. gene_symbol, or variant.name. If the value coming out of the vcf file is an empty string, I was hoping we could get a null.

Reply:
At the moment it will pass through the empty string. I will change it

Reply:
Change made